### ISSUE TITLE
Time controls is not working as expected when the charts tab is visible

### ISSUE THREAD

**Issue by adam-collins** [AtlasOfLivingAustralia/regions#107]
The "Explore by species" tab, and map interaction, including time controls, appear to be working except for
- Selecting a group, e.g. "Birds", then clicking the time control "play" repeatedly toggles the group expansion between open and closed states.
- Selecting a subgroup, e.g. "Doves" under "Birds", does not give any indication that "Doves" is selected. Combined with time control "play", it is unclear if the species column contains all "Birds" (the open group) or the "Doves" selection (the actual filter applied).
- Selecting a species in the species column is a little unclear that it is selected, except for the two buttons beneath the selected species. Clicking the time control "play" and it is not as clear as I would prefer that the species selection no longer applies.

The "Explore by taxonomy" tab
- There is insufficient "reset" taking place when clicking between the tabs.
- Both the "play by decade" and manually setting the time control results in no chart displayed and no date filter applied to points on the map.
- Clicking on the chart to "drill down" does update the map.
- Clicking on the "Previous rank" button below the chart, after "drill down", does update the map.
- The text on the "View records for ..." button appears correct after "drill down" and clicking on "Previous rank".
- The biocache-hub page opened after clicking "View records for ..." button appears correct after "drill down" and clicking on "Previous rank".
- Clicking on the "reset" button in the time controls can result in incorrect labelling of "View records for ...", does not correctly reset the "Previous rank" button below the chart, and 2 resets are required to reset the chart.
- The "Click a slice to drill into the next taxonomic level." button appears to toggle the chart between "by phylum" and "by kingdom". When on the "by phylum" using this, "drill down" incorrectly reported "by kingdom" and broke subsequent "drill down".

Work required
- Reset any chart or table selections, and map display, when switching between tabs.
- Only reset the time controls when clicking the time control reset button.
- Remove the "Click a slice to drill into the next taxonomic level" "toggle" functionality.
- Fix the charts tab map to correctly apply any defined time filter.
- Fix the charts tab chart to correctly apply any defined time filter.
- Change the style of any selected subgroup or species name so that is more clear that that is selected and the map reflects this selection.
- Fix the time control "play" so that it does not toggle the expand and collapse of the selected group filter (when a group filter is selected). The selected group filter should be in open state and be styled differently from the other group names to indicate that it is the active filter.
- In the chart and for the item with no rank value, e.g. `-kingdom:*`, label it so that it does not appear as an error and so it can be clicked. e.g. "unknown".

**Comment by adam-collins** [comment]
Might be nothing, but there appears to be a problem with biocache-service's [breakdown request](https://biocache-ws.ala.org.au/ws/breakdown?q=cl10925%3A%22NORTHERN+TERRITORY%22&fq=rank:(species%20OR%20subspecies)&fq=-occurrence_status_s:absent&fq=geospatial_kosher:true&fq=occurrence_year:*&pageSize=50&fq=&rank=kingdom&name=Bacteria) today.
- Works on test.
- Works with production SOLR.
- Failed unexpectedly in production biocache-service by returning an empty 200 response. Nothing in the logs indicate an issue.

**Comment by adam-collins** [comment]
Should the chart show "no data", it might be this issue https://github.com/AtlasOfLivingAustralia/biocache-service/issues/926

**Comment by kylie-m** [comment]
Hi Adam, I've just completed testing on regions generally, including adding these changes into the testing plan. 
Links: 
- [Updated docs on Confluence](https://confluence.csiro.au/x/EoFKUQ) and 
- [testing notes for this specific release on Teams](https://csiroau.sharepoint.com/:x:/r/sites/AtlasofLivingAustraliaTeams/Shared%20Documents/Teams/Systems/Testing/Regions/regions-test-vx.x.x-Nov2024.xlsx?d=w4952fe56c35441089ed5fe9293fa4d46&csf=1&web=1&e=CYIWFH)

In summary I've tested for the time controls, selection of rank being clearer, removal of the collapse/expand during play, timespan selection is retained when switching between tabs
